server/backend-tests/api.test.js

```diff
@@ -0,0 +1,34 @@
+const supertest = require('supertest');
+const app = require('../server.js');
+
```

Comment:
This file actually looks pretty good to me. I don't see anything here. Good work on this. You have put some real thought into your tests as well.